src/utils/js-yaml-loader.js

```diff
@@ -0,0 +1,150 @@
+'use strict';
```

Comment:
Generally, could we please add jsdoc / comments and get test coverage back to 100%